xrspatial/analytics.py

```diff
@@ -0,0 +1,88 @@
+from xrspatial import slope
+from xrspatial import curvature
+from xrspatial import aspect
+
+import xarray as xr
+
+
+def summarize_terrain(terrain: xr.DataArray):
+    """
+    Calculates slope, aspect, and curvature of an elevation terrain and return a dataset
+    of the computed data.
+
+    Parameters
+    ----------
+    terrain: xarray.DataArray
+        2D NumPy, CuPy, or Dask with NumPy-backed xarray DataArray of elevation values.
+
+    Returns
+    -------
+    summarized_terrain: xarray.Dataset
+        Dataset with slope, aspect, curvature variables with a naming convention of
+        `terrain.name-variable_name`
+
+    Examples
+    --------
+    .. sourcecode:: python
```

Comment:
Please can you put a line of whitespace after this `.. sourcecode:: python` line?

Otherwise when you generate the docs using sphinx it gives a warning. The large number of these warning got in the way of diagnosing the docs build failure earlier in the week, so we should adopt a policy of having the line of whitespace.

Reply:
Thanks Ian, changed as requested!